### PR TITLE
KAFKA-18011: Remove ZooKeeper sections from the docs

### PR DESCRIPTION
**Pull request by mimaison**
The remaining ZooKeeper bits (SCRAM) are to be done in https://issues.apache.org/jira/browse/KAFKA-18012

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: docs/migration.html [hunk 1]
```diff
@@ -1,34 +0,0 @@
-<!--
```
Comment (mimaison) [inline docs/migration.html:1]: As far as I can tell this page is not accessible on the website. Anyway it's about Kafka 0.8!

File: docs/ops.html [hunk 2]
```diff
@@ -1213,41 +1213,7 @@ <h4 class="anchor-heading"><a id="multitenancy-more" class="anchor-link"></a><a
     <strong>Data contracts:</strong> You may need to define data contracts between the producers and the consumers of data in a cluster, using event schemas. This ensures that events written to Kafka can always be read properly again, and prevents malformed or corrupt events being written. The best way to achieve this is to deploy a so-called schema registry alongside the cluster. (Kafka does not include a schema registry, but there are third-party implementations available.) A schema registry manages the event schemas and maps the schemas to topics, so that producers know which topics are accepting which types (schemas) of events, and consumers know how to read and parse events in a topic. Some registry implementations provide further functionality, such as schema evolution, storing a history of all schemas, and schema compatibility settings.
   </p>
 
-
-  <h3 class="anchor-heading"><a id="config" class="anchor-link"></a><a href="#config">6.5 Kafka Configuration</a></h3>
```
Comment (mimaison) [inline docs/ops.html:1217]: This section was pretty much useless, it did not provide the important configuration names and the "example production server configuration" was really incomplete.

File: docs/security.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
 <script id="security-template" type="text/x-handlebars-template">
     <h3 class="anchor-heading"><a id="security_overview" class="anchor-link"></a><a href="#security_overview">7.1 Security Overview</a></h3>
-    In release 0.9.0.0, the Kafka community added a number of features that, used either separately or together, increases security in a Kafka cluster. The following security measures are currently supported:
```
Comment (mimaison) [inline docs/security.html:20]: I think we can remove this outdated sentence.

File: docs/security.html [hunk 4]
```diff
@@ -1561,16 +1524,6 @@ <h4 class="anchor-heading"><a id="security_authz_examples" class="anchor-link"><
             <pre><code class="language-bash">$ bin/kafka-acls.sh --bootstrap-server localhost:9092 --add --allow-principal User:Bob --consumer --topic Test-topic --group Group-1 </code></pre>
             Note that for consumer option we must also specify the consumer group.
             In order to remove a principal from producer or consumer role we just need to pass --remove option. </li>
-
-        <li><b>Admin API based acl management</b><br>
```
Comment (mimaison) [inline docs/security.html:1565]: All the commands above already use `bootstrap-server` so this section is not needed anymore.

**Comment by mimaison** [comment]
@showuon @chia7712 Can you take a look? Thanks

**Review by chia7712 — APPROVED** [review]
LGTM

File: docs/security.html [hunk 5]
```diff
@@ -2373,157 +2326,4 @@ <h3 class="anchor-heading"><a id="security_rolling_upgrade" class="anchor-link">
...
-zookeeper.ssl.client.enable=true
-# required to use TLS to ZooKeeper
-zookeeper.clientCnxnSocket=org.apache.zookeeper.ClientCnxnSocketNetty
-# define trust stores to use TLS to ZooKeeper; ignored unless zookeeper.ssl.client.enable=true
-# no need to set keystore information assuming ssl.clientAuth=none on ZooKeeper
-zookeeper.ssl.truststore.location=/path/to/kafka/truststore.jks
-zookeeper.ssl.truststore.password=kafka-ts-passwd
-# tell broker to create ACLs on znodes (if using SASL authentication, otherwise do not set this)
-zookeeper.set.acl=true</code></pre>
-</script>
```
Comment (chia7712) [inline docs/security.html:2527]: we should keep this ... will file a MINOR to fix it

Comment (chia7712) [reply docs/security.html:2527]: @FrankYang0529 FYI

Comment (FrankYang0529) [reply docs/security.html:2527]: Create a PR for it: https://github.com/apache/kafka/pull/17944

**Pull request merged**